tests/run/datetime_pxd.pyx

```diff
@@ -238,3 +241,39 @@ def test_datetime_attrs_inlined(datetime dt):
         dt.second,
         dt.microsecond,
     )
+
+def test_date_from_timestamp():
+    """
+    >>> from datetime import datetime
+    >>> tp, dt = test_date_from_timestamp()
+    >>> tp == dt
+    True
+    """
+    tp = date_from_timestamp(1518185542)
+    dt = py_datetime.date(2018, 2, 9)
+    return tp, dt
+
+def test_get_utc():
+    """
+    >>> from datetime import datetime
+    >>> test_get_utc()
+    True
+    """
+    try:
+        utc = get_utc()
+    except RuntimeError:
+        if sys.version_info >= (3, 7):
+            raise  # get_utc() is only supposed to raise on Python < 3.7
+    return True
+
+def test_datetime_from_timestamp():
+    """
+    >>> from datetime import datetime
+    >>> tp, dt = test_datetime_from_timestamp()
+    >>> tp == dt
+    True
+    """
+    time = py_time.time()
+    tp = datetime_from_timestamp(time)
+    dt = py_datetime.datetime.from_timestamp(time)
```

Comment:
Looks like no-one has actually run these tests before…
```suggestion
    dt = py_datetime.datetime.fromtimestamp(time)
```

Reply:
That's embarrassing 😬

I've played around with the test flags so that I don't need to run _all_ tests, which ones do you use to only run these tests?

Reply:
You can pass a regex for the test name as argument, e.g. `python3 runtests.py --backend=c -vv datetime`.

Reply:
Perfect! Thank you.